reproject/hips/core.py

```diff
@@ -403,37 +404,21 @@ def determine_healpix_level(wcs_in, max_level=25):
     ----------
     wcs_in : `~astropy.wcs.WCS`
         The WCS of the input array
-    max_level : int, optional
-        The maximum level to consider. Default is 25, corresponding to 6 mas.
-        Can be overridden, but included as as hint to users that images could
-        get really huge at this level, as it contains 10^16 pixels over the sky
+    tile_size : int
+        The size of the tile in pixels
 
     Returns
     -------
     level : int
         The recommended HEALPix level
     """
 
-    # Get the pixel scale from the input WCS in degrees
-    # We use the geometric mean of the pixel scales in both dimensions
+    # Get the pixel scale from the input WCS
     pixel_scale = wcs_in.proj_plane_pixel_area()
```

Comment:
Maybe call this ``pixel_area``

Also we should probably use an APE 14 friendly approach here ideally, but keep it as-is for now, and once we merge the PR in I can work on generalizing it - we have some code in ``find_optimal_celestial_wcs`` that deals with this so maybe this should be factored out into a helper function we can use here.